### DISCUSSION TITLE
SEB doesn't start (just "close" and "uninstall")

### DISCUSSION THREAD

**Discussion by KKS-Support** [SafeExamBrowser/seb-win-refactoring#582] [Q&A]
We are using SEB in combination with the exam platform "classtime.com". 
One you enter the classtime-"session code" and click on "Herunterladen" (=download), SEB opens but just with the two options "uninstall" and "close". 

![seb_error](https://user-images.githubusercontent.com/128165897/225948052-1e6568a3-1af2-4b8a-8a2d-07e6ef9e0fe7.JPG)

We tried already: Uninstalling SEB and reinstalling. Unistalling and Deleting the folders in C:\users\...\appdata\local and ...\roaming and reinstalling SEB. Deleting browser history/temp. files. 

It's just not working on this specific laptop... on other devices it runs perfect. Maybe anybody can help? Thank you in advance.

OS: Windows 11 Home 22H2
SEB: 3.4.1.505
Webview: 111.0.1661.44

**Comment by dbuechel** [comment]
What opens here is not SEB but the installer of SEB. There most likely is a configuration issue on that system, unless the download button does not start SEB but keeps downloading the SEB installer? Either way, you maybe can resolve the issue by manually uninstalling, restarting and then reinstalling the latest release version of SEB.

**Comment by KKS-Support** [comment]
Thanks, dbuechel for your answer. 
When I click on the "download" button it loads a SEB-config-File and when I open it, the installer of SEB starts and as you said, not SEB itself. I also uninstalled SEB (and Edge WebView) and reinstalled it. Unfortunately without solving the problem. 
But you now gave me an idea: maybe the SEB config files have a wrong file association on this specific system... I will check this.
